qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -154,6 +154,11 @@ def run(self, dag):
             if len(cm_graph) == len(im_graph):
                 chosen_layout = layout
                 break
+            # If there is no error map avilable we can just skip the scoring stage as there
+            # is nothing to score with so any match is the best we can find.
```

Comment:
```suggestion
            # is nothing to score with, so any match is the best we can find.
```

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8978/commits/468a08b4a3f4c2aaf36898077559444d4b5db76b